inc/languages/english/usercp.lang.php

```diff
@@ -299,6 +299,7 @@
 $l['attachstats_quota'] = "Attachment Quota";
 $l['attachstats_totaldl'] = "Total Downloads";
 $l['attachstats_bandwidth'] = "Approximate Bandwidth Usage";
+$l['error_avatarurltoolong'] = "Sorry, but we cannot change your avatar as the new avatar URL specified is too long. You can try to use a shorter URL which doesn't exceed the database field length.";
```

Comment:
I think this is better worded without mentioning a DB field as that could confuse some users:

```suggestion
$l['error_avatarurltoolong'] = "Sorry, but we cannot change your avatar as the new avatar URL specified is too long. You can try to use a shorter URL which is less than 200 characters long.";
```